src/poll/poll.dtos.ts

```diff
@@ -166,7 +185,9 @@ export class PollResponseDto {
     })
         strategy_config: StrategyConfig[];
 
-    @ArrayMaxSize(8)
+    // TODO: Bot can only have 5 buttons per ActionRow. Supporting > 5 will require bot code refactor
```

Comment:
Honestly I think if there are any more than 5, we would ask the user to vote in the FE. Future work.